engine/src/main/scala/cromwell/engine/callexecution/BackendCallExecutionActor.scala

```diff
@@ -5,26 +5,27 @@ import cromwell.engine.backend._
 import cromwell.engine.callexecution.CallExecutionActor._
 import cromwell.logging.WorkflowLogger
 
+import scala.concurrent.ExecutionContext
 import scala.language.postfixOps
 
 /**
   * Actor to manage the execution of a single backend call.
   * */
-class BackendCallExecutionActor(backendCall: BackendCall) extends CallExecutionActor {
+class BackendCallExecutionActor(jobDescriptor: BackendCallJobDescriptor) extends CallExecutionActor {
   override val logger = WorkflowLogger(
     this.getClass.getSimpleName,
-    backendCall.workflowDescriptor,
+    jobDescriptor.workflowDescriptor,
     akkaLogger = Option(akkaLogger),
-    callTag = Option(backendCall.key.tag)
+    callTag = Option(jobDescriptor.key.tag)
   )
 
-  override val call = backendCall.call
-  override def poll(handle: ExecutionHandle) = backendCall.poll(handle)
-  override def execute(mode: ExecutionMode) = mode match {
-    case Execute => backendCall.execute
-    case Resume(jobKey) => backendCall.resume(jobKey)
-    case UseCachedCall(cachedBackendCall) => backendCall.useCachedCall(cachedBackendCall)
+  override val call = jobDescriptor.key.scope
```

Comment:
may need to do a replace of `key.scope` with `call` to pick up your spiffy new method